fluent-bundle/src/bundle.rs

```diff
@@ -21,6 +21,12 @@ use crate::types::FluentValue;
 
 #[derive(Debug, PartialEq)]
 pub struct Message<'m> {
```

Comment:
Would it make sense to call this `RawMessage`?

Reply:
I don't think so. We have `FluentBundle::get_message`, so we return a `Message`. There's no other `Message` in the crate.

Reply:
how about `FluentMessage`, then? I'm concerned about the naming conflict with `ast::Message`. I understand that it's not really a problem in code, because the module namespacing works to our advantage. But conceptually, I'd prefer for these two message to have distinct names.

Reply:
Yep, filed an issue!